module/Application/view/layout/layout.phtml

```diff
@@ -106,13 +106,9 @@ $(document).ready(function() {
                         <?php if (null !== ($identity = $this->identity())): ?>
                             <?php
                             if ($identity instanceof User) {
-                                $name = $identity->getLogin();
+                                $name = $identity->getName();
                             } else {
-                                if (1 === preg_match('/^((?:a|m)(?:[0-9]{4,5}))@GEWISWG\.GEWIS\.NL$/', $identity, $matches)) {
-                                    $name = $matches[1];
-                                } else {
-                                    $name = $this->translate('User');
-                                }
+                                $name = $this->translate('User');
                             }
```

Comment:
Technically the `else` is no longer needed right? As if you are authenticated on this page (i.e. you are already in the currently flow because `null !== $this->identity()`) you are guaranteed to be a `User`.

Reply:
Changed in cf31568f96f9bd9024fa55ba5e56b1440ff28162 where we now assume that the current identity always is a User object